README.md

```diff
@@ -99,6 +100,50 @@ All pros of IPython Notebook and Markdown, no cons!
   Hello world
   ```
 
+* **Notebook metadata** can be specified in [YAML](http://yaml.org/) inside
+  Jekyll-style [front-matter](http://jekyllrb.com/docs/frontmatter/) dashes
+  at the beginning of a document:
+
+  ```markdown
+  ---
+  kernelspec:
+    name: some-non-native-kernel
+  ---
+
+  First cell content
+  ```
+
+  Native kernel metadata will be elided by default: non-python kernels haven't
+  been tested yet, but support is planned.
+
+* **Cell metadata** is specified with YAML stream documents with dashes and
+  periods, such as to create slides:
+
+  ```markdown
+  # Previous slide
+
+  ---
+  slideshow:
+    slide_type: slide
+  ...
+
+  # Some Slide Content
+  ```
+
+  > NOTE: You probably shouldn't use `---` to mean an `<hr/>`: `***`
+  could be a suitable substitute.
+
+* Null metadata (i.e. splitting a markdown cell) can be created with just
+  three dashes. This is useful when adding slideshow notes or skipped cells.
+
+  ```markdown
+  A cell
+
+  ---
```

Comment:
Not sure I totally understand this. Does `---` mean that the next cell has a special metadata field named `{'ipymd': {'empty_meta': True}}` and that is should be skipped by converters or something?